src/open_inwoner/accounts/templates/registration/login.html

```diff
@@ -52,10 +52,7 @@ <h1 class="utrecht-heading-1">{% trans 'Welkom' %}</h1>
                                     {% endrender_card %}
                                 {% endif %}
                             {% endrender_column %}
-                        {% endif %}
-                        {# end digid #}
 
-                        {% if settings.DIGID_ENABLED %}
                             {% get_solo 'mozilla_django_oidc_db.OpenIDConnectConfig' as oidc_config %}
                             {% get_solo 'configurations.SiteConfiguration' as site_config %}
                             {% if oidc_config.enabled and site_config.openid_enabled_for_regular_users %}
```

Comment:
@stevenbal  Here is yet another OpenID connect statement, inside DIGID_ENABLED. So there are actually two.